### PR TITLE
Helm chart update 

### PR DESCRIPTION
**Pull request by FR-Solution**
Update of helm charts with a detailed description of the installation in vanilla k8s.

### CODE REVIEW

File: charts/yandex-csi-controller/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+# Install
+
+Read the k8s-csi-controller-key.json file and load the content instead of the example in the block <service-account-json>
+
+```bash
+
+export NAMESPACE=kube-system
+
+cat <<EOF | kubectl apply -f -
```
Comment (zuzzas) [inline charts/yandex-csi-controller/README.md]: Can't we just pass this as a `--set` to Helm chart?

File: charts/yandex-csi-controller/templates/storage-classes.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: yc-network-hdd
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
```
Comment (zuzzas) [inline charts/yandex-csi-controller/templates/storage-classes.yaml:8]: That seems like a candidate for parametrization. With a "default" value set to `yc-network-hdd`.

File: charts/yandex-csi-controller/values.yaml [hunk 3]
```diff
@@ -0,0 +1,172 @@
+
+ycsi:
+  cloud_secret:
```
Comment (zuzzas) [inline charts/yandex-csi-controller/values.yaml]: Let's just provide this as a string blob that contains the whole ServiceAccount JSON. No reason to split it whatsoever.

File: charts/yandex-csi-controller/values.yaml [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+    limits:
+      cpu: 1000m
+      memory: 1Gi
+    requests:
+      ephemeral-storage: 50Mi
+      cpu: 100m
+      memory: 128Mi
+
+
+  securityContext:
```
Comment (zuzzas) [inline charts/yandex-csi-controller/values.yaml:129]: Please, hard-code securityContext and remove it from `values`. These values are integral to the workings of these containers, there should be no reason for a user to change them.

**Review by zuzzas — CHANGES_REQUESTED** [review]
Thanks! Overall, looks great, but there are a couple of points that I'd like to be addressed.

File: charts/yandex-csi-controller/templates/storage-classes.yaml (hunk 2 above)
Comment (FR-Solution) [reply charts/yandex-csi-controller/templates/storage-classes.yaml:8]: fix it

File: charts/yandex-csi-controller/README.md (hunk 1 above)
Comment (FR-Solution) [reply charts/yandex-csi-controller/README.md]: fix it

File: charts/yandex-csi-controller/values.yaml (hunk 3 above)
Comment (FR-Solution) [reply charts/yandex-csi-controller/values.yaml]: fix it

File: charts/yandex-csi-controller/values.yaml (hunk 4 above)
Comment (FR-Solution) [reply charts/yandex-csi-controller/values.yaml:129]: I still want to leave this field parametrized

**Comment by zuzzas** [comment]
Thanks!

**Review by zuzzas — APPROVED** [review]

**Pull request merged**